simtools/model/camera.py

```diff
@@ -505,7 +507,7 @@ def _calcEdgePixels(self, pixels, neighbours):
 
         edgePixelIndices = list()
 
-        for i_pix, xyPixPos in enumerate(zip(pixels["x"], pixels["y"])):
+        for i_pix, _ in enumerate(zip(pixels["x"], pixels["y"])):
```

Comment:
Since the positions are not used in this for loop (this is an excerpt from a larger code), the for loop should change to be `for i_pix in range(len(pixels["x"])):`

Reply:
OK - aa735c8